examples/cpu_render.rs

```diff
@@ -75,10 +66,17 @@ fn main() {
         angle = angle + velos.x / 10.;
 
         // Set the viewport
-        let origin = V3c::new(angle.sin() * radius, radius, angle.cos() * radius);
         let viewport_ray = Ray {
-            direction: (V3c::unit(0.) - origin).normalized(),
-            origin,
+            origin: V3c {
```

Comment:
Wow, how did these values come to life?

Reply:
I printed out the viewport from the minecraft example, plugged it into CPU example, so I've had a way to confirm if the behavior is the same on the GPU and CPU.. Because of this I was able to figure out this: https://github.com/davids91/shocovox/commit/456aed99d1579ac1b376b8684b8625b27acbe031